src/CMakeLists.txt

```diff
@@ -165,6 +166,7 @@ if(Omega_h_USE_SimModSuite)
   set(CMAKE_MODULE_PATH ${OLD_CMAKE_MODULE_PATH})
   set(Omega_h_SOURCES ${Omega_h_SOURCES} Omega_h_meshsim.cpp)
   set(Omega_h_SOURCES ${Omega_h_SOURCES} Omega_h_matchMeshsim.cpp)
+  set(Omega_h_SOURCES ${Omega_h_SOURCES} Omega_h_simModel2d.cpp)
```

Comment:
We should probably consider using `list(APPEND Omega_h_sources ...)` (https://cmake.org/cmake/help/latest/command/list.html#append) before we end up with a bunch of calls to `set`. There are other places that can benefit from this too

Reply:
I agree.  Thank you.  Draft PR #97 addresses this.